src/umpire/interface/c_fortran/wrapResourceManager.cpp

```diff
@@ -1,22 +1,15 @@
 // wrapResourceManager.cpp
-// This file is generated by Shroud 0.11.0. Do not edit.
+// This is generated code, do not edit
 // Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
 // project contributors. See the COPYRIGHT file for details.
 //
 // SPDX-License-Identifier: (MIT)
 #include "wrapResourceManager.h"
+#include <stdlib.h>
 #include <string>
 #include "umpire/Allocator.hpp"
 #include "umpire/ResourceManager.hpp"
 
-#include "umpire/strategy/AllocationAdvisor.hpp"
```

Comment:
@ltaylor16 - I'm not sure why these lines get removed, but now this fails to compile...